pages/1.11/installing/oss/upgrading/index.md

```diff
@@ -1,153 +0,0 @@
----
-layout: layout.pug
-navigationTitle:  Upgrading
-excerpt: Upgrading a DC/OS cluster
-title: Upgrading
-menuWeight: 4
----
-
-An upgrade is the process of adding new features, replacing the existing features with new features/functionality or adding a major configuration change. You can upgrade DC/OS only if you have used the installation process to install DC/OS on your cluster.
```

Comment:
Change the first sentence to: 
An upgrade is the process of moving between major releases to add new features, replace existing features with new features/functionality or performing a major configuration change.